src/task1/projection-module/projection.py

```diff
@@ -53,7 +53,7 @@ def get_pixel_coords(self, pimg, yawangle, pitchangle, rollangle, latdrone, long
             rollangle (float): The roll angle in degrees, for axis 1
             latdrone (float): the latitude of the drone in degrees
             longdrone (float): the longditude of the drone in degrees
-            altdrone (float): the altitude of the drone in meters
+            altdrone (float): the absolute altitude of the drone in meters

```

Comment:
Don't use the term absolute since this could be interpreted as either AGL(Above ground level) or AMSL (Above mean sea level)